dashboard/legacy/middleware/helpers/projects.rb

```diff
@@ -137,10 +137,8 @@ def publish(channel_id, type, user)
     raise NotFound, "channel `#{channel_id}` not found" if project_query_result.empty?
 
     rails_project = get_rails_project(project_id)
-    if rails_project.apply_project_age_publish_limits?
```

Comment:
We are removing these exceptions:
https://github.com/code-dot-org/code-dot-org/blob/f9ac6acef493ab6df7be31602b875c445906004d/dashboard/app/models/project.rb#L57-L66
according to [this comment](https://docs.google.com/document/d/1ULRkKKrbMgGHA10XlPUvDaBA7wBELbHcTECvD-NYfpI/edit?disco=AAABRarQ9KQ) in the product spec.